src/server.rs

```diff
@@ -76,13 +74,21 @@ impl Server {
     }
 
     fn get_delta_time(&mut self) -> f32 {
+        if self.slowtime_is_active() {
+            constants::DELTA_TIME / constants::POWERUP_SLOWTIME_FACTOR;
+        }
+
+        constants::DELTA_TIME
+    }
+
+    fn slowtime_is_active(&mut self) -> bool {
         for player in &mut self.state.players {
             if player.powerups.iter().any(|powerup|powerup.kind == PowerUpKind::SlowTime) {
-                return 1./300.;
+                true;
```

Comment:
If you put a semicolon after the value you want to return (without expicitly writing return), it won't return it, right @TheZoq2 ?

Reply:
No it won't

Reply:
Then this should be fixed.

Reply:
It looks like `mut` could be removed also

Reply:
Yes, and then `mut` can be removed from get_delta_time as well